support/common/src/main/kotlin/com/studentcenter/weave/support/common/vo/UpdateParam.kt

```diff
@@ -0,0 +1,10 @@
+package com.studentcenter.weave.support.common.vo
+
+data class UpdateParam<T>(
+    val value: T?,
+)
+
+fun <T> UpdateParam<T>?.update(oldValue: T?, newValue: T?): T? {
+    return if(this == null) oldValue else newValue
+}
```

Comment:
p3) oldbiew만 받아도 newbie는 자기가 알고 있어서 괜찮을 것 같아요!
UpdateValue를 가져오는 느낌이라 이름도 바꾸면 좋을 것 같네요

```suggestion
fun <T> UpdateParam<T>?.getUpdateValue(oldValue: T?): T? {
    return if(this == null) oldValue else this.value
}
```